### ISSUE TITLE
Specifying HeightRequest in Webview when wrapped by ScrollView set "invisible" causes crash in iOS

### ISSUE THREAD

**Issue by bobwhitten** [dotnet/maui#26795]
### Description

e.g., Add this to MAUI template project:
```
            <ScrollView x:Name="myMediaFrame" 
                IsVisible="False"
                >
                <WebView x:Name="myWebView" 
                    HeightRequest="1000"
                    />
            </ScrollView>
```
This results in this on startup (in my app, when the page appeared):

ObjCRuntime.ObjCException
  Message=Objective-C exception thrown.  Name: CALayerInvalidGeometry Reason: CALayer position contains NaN: [22 nan]. Layer: <CALayer:0x30034e780; position = CGPoint (0 0); bounds = CGRect (0 0; 0 0); delegate = <Microsoft_Maui_Platform_MauiWKWebView: 0x11640b000; frame = (0 0; 0 0); backgroundColor = UIExtendedGrayColorSpace 0 0; layer = <CALayer: 0x30034e780>>; sublayers = (<CALayer: 0x300376cc0>); allowsGroupOpacity = YES; backgroundColor = <CGColor 0x30266d9a0> [<CGColorSpace 0x30211e580> (kCGColorSpaceICCBased; kCGColorSpaceModelMonochrome; Generic Gray Gamma 2.2 Profile; extended range)] ( 0 0 )>

Removing EITHER IsVisible="False" from VerticalStackLayout OR HeightRequest="1000" from WebView prevents crash.

This behavior did not happen in Windows.
This particular behaviour does not slow me down, but it was certainly unexpected, and took some time to find.

### Steps to Reproduce

Run reproduction project. Observe crash. 
Modify as noted above, observe no crash.

### Link to public reproduction project repository

https://github.com/bobwhitten/DemoWebView

### Version with bug

9.0.21 SR2.1

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iPadOS 18.2, 17.5.1

### Did you find any workaround?

Either do not start Invisible, or do not set height request.

### Relevant log output

```shell

```

**Comment by Ying-6** [comment]
Hi @bobwhitten I can't open your project, could you please relink the public project? Looking forward to your reply!

**Comment by Ying-6** [comment]
Due to the project linked by the user cannot be opened. This issue has been verified using Visual Studio 17.13.0 Preview 2.1 (9.0.21 & 9.0.14 & 8.0.100) on MAUI template project. Can repro this issue on iOS platform.

**Comment by bobwhitten** [comment]
Apologies, the GitHub proj was private.  It is now public.  Thanks to Ying-6 for pointing this out.

**Comment by PureWeen** [comment]
I'm curious if this will be resolved by
https://github.com/dotnet/maui/pull/26763

**Comment by bhavanesh2001** [comment]
@PureWeen  Seems like this is fixed with #26763. Can reproduce issue with 9.0.21 but not with 9.0.30 or latest.

**Comment by jasells** [comment]
I am seeing a similar (random) issue with Maui 8.0.100 in production code, iOS only, all devices running iOS 18.x:

```
CALayerInvalidGeometry Reason: CALayer position contains NaN: [9.75 nan]. Layer: <CALayer:0x303a53fc0; position = CGPoint (9.75 inf); bounds = CGRect (0 0; 19.5 inf);
 delegate = <Microsoft_Maui_Platform_LayoutView:
```

Don't know if related to this issue, but seems like a similar issue to past/closed issues:

https://github.com/dotnet/maui/issues/19120
https://github.com/dotnet/maui/issues/5346
https://github.com/dotnet/maui/issues/5749
https://github.com/dotnet/maui/pull/5733

**Comment by jasells** [comment]
Is _potential_ fix https://github.com/dotnet/maui/pull/26763 also available in Maui 8 build?  Which release or is it planned?

**Comment by bhavanesh2001** [comment]
@jasells I don't think this will make it into .NET 8, since support for .NET 8 is ending very soon.
https://dotnet.microsoft.com/en-us/platform/support/policy/maui

**Comment by jasells** [comment]
End of support is still > 6 months away, so I would expect at least 2 more service releases of Maui8....  But I get your point that this might not make it into those.  That's why I asked, one can hope.

**Comment by bhavanesh2001** [comment]
I think you’re confusing the `Patch Release Date` with the `End of Support`—unless you have a time machine? 🤨


<img width="1360" alt="Image" src="https://github.com/user-attachments/assets/3dc8421a-a906-4f8d-b65b-a82519b5d33f" />
